examples/rust/src/bin/client.rs

```diff
@@ -524,7 +524,8 @@ async fn geyser_subscribe(
     request: SubscribeRequest,
     resub: usize,
 ) -> anyhow::Result<()> {
-    let (mut subscribe_tx, mut stream) = client.subscribe_with_request(Some(request)).await?;
+    let (mut subscribe_tx, mut stream) =
+        client.subscribe_with_request(Some(request.clone())).await?;
```

Comment:
```suggestion
        client.subscribe_with_request(Some(request)).await?;
```

`.clone()` is not required now

Reply:
updated